ydb/docs/en/core/reference/ydb-cli/workload-tpch.md

```diff
@@ -1,71 +1,96 @@
+
+```bash
+{{ ydb-cli }} workload tpch --path tpch/s1 ...
+```
+
+### Available options {#common_options}
+
+Option name | Option description
+---|---
+`--path` or `-p` | Path to the directory with tables. Default value is `/`
+
+## Initializing a load test { #init }
 
 Before running the benchmark, create a table:
 
 ```bash
-{{ ydb-cli }} workload tpch init
+{{ ydb-cli }} workload tpch --path tpch/s1 init
 ```
 
-See the description of the command to run the data load:
+See the description of the command for initializing tables:
 
 ```bash
 {{ ydb-cli }} workload tpch init --help
 ```
 
-### Available parameters {#init-options}
+{% include [init_options](./_includes/workload/init_options_tpc.md) %}
 
-| Option name | Option description |
----|---
-| `--path <value>` | Directory to create tables in. Default value - empty string. |
-| `--store <value>` | Type of table storage. Acceptable values: `row`, `column`. Default value: `row`. |
+## Loading data into a table { #load }
 
-## Uploading data to the table {#load}
+Load data into a table. The data will be generated directly by {{ ydb-cli }}:
+
+```bash
+{{ ydb-cli }} workload tpch --path tpch/s1 import generator --scale 1
+```
 
-You can download the dataset generator for the TPC-H benchmark by the [link](http://tpc.org/tpc_documents_current_versions/current_specifications5.asp).
-Then follow the instructions from README.
-In the `dss.h` file, you can specify a field separator: Default: `#define SEPARATOR '|'`.
-In the sample data upload script, `'\t'` is used as a separator.
+See the description of the command to load the data:
 
 ```bash
-for table in region nation supplier customer part partsupp orders lineitem; do
-    echo "Start data load to $table"
-    {{ ydb-cli }} import file tsv --header --path "$table" --input-file $table.tsv --newline-delimited
-    echo "Finish data load to $table"
-done
+{{ ydb-cli }} workload tpch import --help
 ```
 
-## Running a load test {#run}
+### Available options { #load_files_options }
+
+Option name | Option description
+---|---
+`--scale <value>` | Data scale. Powers of ten are usually used.
+`--tables <value>` | Comma-separated list of tables to generate. Available tables: `customer`, `nation`, `order_line`, `part_psupp`, `region`, `supplier`. All tables are created by default.
+`--proccess-count <value>` or `-C <value>` | Data generation can be split into several processes, this parameter specifies the number of processes. Default value: 1.
+`--proccess-index <value>` or `-i <value>` | Data generation can be split into several processes, this parameter specifies the process number. Default value: 0.
+`--state <path>` | Path to the download state file. If the download was interrupted for some reason, the download will be continued from the same place when it is started again.
+`--clear-state` | Relevant if the `--state` parameter is specified. Clear the state file and start the download from the beginning.
```

Comment:
```suggestion
| Name                                  | Description                                                                                                                                                  | Default value |
|---------------------------------------|--------------------------------------------------------------------------------------------------------------------------------------------------------------|---------------|
| `--scale <value>`                     | Data scale. Typically, powers of ten are used.                                                                                                                |               |
| `--tables <value>`                    | Comma-separated list of tables to generate. Available tables: `customer`, `nation`, `order_line`, `part_psupp`, `region`, `supplier`.                         | All tables    |
| `--process-count <value>` or `-C <value>` | Specifies the number of processes for parallel data generation.                                                                                                | `1`           |
| `--process-index <value>` or `-i <value>` | Specifies the current process number when data generation is split into multiple processes.                                                                            | `0`           |
| `--state <path>`                      | Path to the state file for resuming downloads. If the download is interrupted, it will continue from the same point when restarted.                            |               |
| `--clear-state`                       | Relevant if the `--state` parameter is specified. Clears the state file and starts the download from the beginning.                                             |               |
```